internal/v2/service/delegation.go

```diff
@@ -103,13 +104,37 @@ func FromDelegationDocument(delegation indexerdbmodel.IndexerDelegationDetails)
 			},
 		},
 		State:                    state,
-		CanExpand:                delegation.CanExpand,
+		CanExpand:                false, // Will be set by runtime evaluation below
```

Comment:
Maybe it's better just to remove this line since it will have zero value (false) anyway